from PyQt5 import QtCore, QtGui, QtWidgets
import resources

class Ui_Form(object):
    def setupUi(self, Form):
        self.horizontalLayout_2 = QHBoxLayout()
        self.horizontalLayout_2.setObjectName(u"horizontalLayout_2")
        self.label_7 = QLabel(Form)
        self.label_7.setObjectName(u"label_7")
        self.horizontalLayout_2.addWidget(self.label_7)
        self.label_2 = QLabel(Form)
        self.label_2.setObjectName(u"label_2")
        self.label_2.setMinimumSize(QSize(30, 35))
        self.label_2.setMaximumSize(QSize(30, 35))
        self.label_2.setPixmap(QPixmap(u":/binance/binance-coin-bnb-logo.png"))
        self.label_2.setScaledContents(True)
        self.horizontalLayout_2.addWidget(self.label_2)
        self.label = QLabel(Form)
        self.label.setObjectName(u"label")
        self.horizontalLayout_2.addWidget(self.label)
        self.label_4 = QLabel(Form)
        self.label_4.setObjectName(u"label_4")
        self.label_4.setTextFormat(Qt.AutoText)
        self.horizontalLayout_2.addWidget(self.label_4)
        self.label_8 = QLabel(Form)
        self.label_8.setObjectName(u"label_8")
        self.label_5 = QLabel(Form)
        self.label_5.setObjectName(u"label_5")
        self.horizontalLayout_2.addWidget(self.label_5)
        self.label_9 = QLabel(Form)
        self.label_9.setObjectName(u"label_9")
        self.horizontalLayout_2.addWidget(self.label_9)
        self.label_6 = QLabel(Form)
        self.label_6.setObjectName(u"label_6")
        self.horizontalLayout_2.addWidget(self.label_6)
        self.label_11 = QLabel(Form)
        self.label_11.setObjectName(u"label_11")
        self.horizontalLayout_2.addWidget(self.label_11)
        self.label_10 = QLabel(Form)
        self.label_10.setObjectName(u"label_10")
        self.horizontalLayout_2.addWidget(self.label_10)
        self.horizontalSpacer = QSpacerItem(40, 20, QSizePolicy.Expanding, QSizePolicy.Minimum)
        self.horizontalLayout_2.addItem(self.horizontalSpacer)
        self.label_12 = QLabel(Form)
        self.label_12.setObjectName(u"label_12")
        self.horizontalLayout_2.addWidget(self.label_12)
        self.label_3 = QLabel(Form)
        self.label_3.setObjectName(u"label_3")
        self.horizontalLayout_2.addWidget(self.label_3)
        self.retranslateUi(Form)

        QMetaObject.connectSlotsByName(Form)
    # setupUi

    def retranslateUi(self, Form):
        Form.setWindowTitle(QCoreApplication.translate("Form", u"Form", None))
        self.label_7.setText("")
        self.label_2.setText("")
        self.label.setText(QCoreApplication.translate("Form", u"<html><head/><body><p><span style=\" font-size:14pt;\">Binance </span><span style=\" font-size:12pt; vertical-align:super;\">Futures</span></p></body></html>", None))
        self.label_4.setText(QCoreApplication.translate("Form", u"name", None))
        self.label_8.setText("")
        self.label_5.setText(QCoreApplication.translate("Form", u"\u041f\u043e\u0434\u043a\u043b\u044e\u0447\u0435\u043d\u0430", None))
        self.label_9.setText("")
        self.label_6.setText(QCoreApplication.translate("Form", u"\u0420\u0435\u0434\u0430\u043a\u0442\u0438\u0440\u043e\u0432\u0430\u0442\u044c", None))
        self.label_11.setText("")
        self.label_10.setText(QCoreApplication.translate("Form", u"\u0423\u0434\u0430\u043b\u0438\u0442\u044c", None))
        self.label_12.setText(QCoreApplication.translate("Form", u"\u0411\u0430\u043b\u0430\u043d\u0441:", None))
        self.label_3.setText(QCoreApplication.translate("Form", u"$999999.99", None))
    # retranslateUi